analysis/active_analyses.R

```diff
@@ -551,7 +552,8 @@ for (c in cohorts) {
                          strata = strata,
                          covariate_sex = covariate_sex,
                          covariate_age = covariate_age,
-                         covariate_other = gsub("cov_bin_history_any_dementia;","",all_covars),
+                         covariate_other = #"cov_cat_ethnicity;cov_cat_deprivation;cov_cat_smoking_status;cov_bin_carehome_status;cov_num_consulation_rate;cov_bin_healthcare_worker;cov_bin_liver_disease;cov_bin_ckd;cov_bin_cancer;cov_bin_obesity;cov_bin_copd;cov_bin_ami;cov_bin_isch;cov_bin_history_mnd;cov_bin_history_migraine;cov_bin_history_parkinson;cov_bin_history_ms;cov_bin_history_parkinson_risk",#;sub_bin_high_vascular_risk",
+                         gsub("cov_bin_hypertension;|cov_bin_diabetes;|cov_bin_history_alzheimer_disease;|cov_bin_history_vascular_dementia;|cov_bin_history_any_dementia;|cov_bin_history_lewy_body_dementia;|cov_bin_history_any_dementia;|cov_bin_history_cog_imp_sympt;","",all_covars),
```

Comment:
Only `cov_bin_hypertension` and `cov_bin_diabetes` need to be removed from this model as these are the only variables used to define the subgroup.